src/workerd/api/container.h

```diff
@@ -26,10 +26,11 @@ class Container: public jsg::Object {
   struct StartupOptions {
     jsg::Optional<kj::Array<kj::String>> entrypoint;
     bool enableInternet = false;
+    jsg::Optional<kj::Array<kj::String>> environmentVariables;
```

Comment:
I think this name should be shortened to `environ` or `env`. I'm ambivalent about which one. Is this the same concept as Workers env, or not? It does seem like the same concept as `process.env` from node, so I guess that leans towards `env`.

I think the format should be an object, like:

```
start({
  env: {
    "PATH": "/usr/bin:/bin",
    "FOO": "bar",
  }
});
```

I suppose we'd need to throw an exception if the name contains `=`, or if either name or value contains `\0`. (Are any other characters technically invalid to include in an env var?)

Reply:
To accept the object format, use `jsg::Optional<jsg::Dict<kj::String>>` as the type.

Reply:
completely agreed on the JS API changes!